sourcemash/frontend/templates/feed.tpl

```diff
@@ -1,6 +1,7 @@
 <div class="container">
   <div class="row valign-wrapper">
-      <h2 class="col s8 m9"><%= model.get('title') %></h2>
+      <h2 class="col s6 m9"><%= model.get('title') %></h2>
+      <span class="col s2 grey-text text-lighten-1">[<%=model.get('item_count')%> items]</span>
```

Comment:
It still seems weird not to list the number of items anywhere. We can get away with it at the category level since there are never that many items in a category, but feeds range from 10 to 90 items so it's just kinda odd not to report that.